### ISSUE TITLE
[Bug] The namespaceSelector element is placed incorrectly in the example ServiceMonitor

### ISSUE THREAD

**Issue by asavtch** [strimzi/strimzi-kafka-operator#1957]
**Description**
In  strimzi-kafka-operator/metrics/examples/prometheus/install/strimzi-service-monitor.yaml the namespaceSelector is placed inside the endpoints element and is therefore not recognized by Prometheus when building the scrape job configs. The bug shows up when trying to set up cross-namespace monitoring of Kafka.

**Expected behavior**
1. The namespaceSelector is placed inside the spec element like so:
```
spec:
  namespaceSelector:
    ...
  selector:
    matchLabels:
      strimzi.io/kind: Kafka
  ...
```
2. The corresponding namespace spec is present in the scrape job config upon creating the ServiceMonitor.

**Comment by scholzj** [comment]
@stanlyDoge @ppatierno Could you please have a look at this?

**Comment by stanlyDoge** [comment]
Thanks @asavtch. Addressed in https://github.com/strimzi/strimzi-kafka-operator/pull/1960

**Comment by stanlyDoge** [comment]
Fixed by #1960
